src/lib.rs

```diff
@@ -848,9 +848,9 @@ pub fn parse_path(path: &PathBuf, config: ParseConfig) -> anyhow::Result<ParseOu
         // Generate HTML for each directory name
         for directory_name in &directory_names {
             let pre_grad_graph_content =
-                get_file_content(&output, "inductor_pre_grad_graph", directory_name);
```

Comment:
Need to keep the old names for backward compatibility.

Reply:
> Need to keep the old names for backward compatibility.

@desertfire I'll add back the file name to be BC. But I'm updating the doc to say it's only compatible with the nightly build in  https://github.com/pytorch/pytorch/pull/154062 because it may still not be compatible with earlier versions of PyTorch (e.g. a few months ago).

Reply:
@desertfire added back in PR https://github.com/pytorch/tlparse/pull/111